packages/app/src/Pages/Login.tsx

```diff
@@ -279,12 +279,20 @@ export default function LoginPage() {
           <div className="flex">
             <input
               dir="auto"
-              type="text"
+              type={key?.startsWith("nsec") ? "password" : "text"}
               placeholder={formatMessage(messages.KeyPlaceholder)}
               className="f-grow"
               onChange={e => setKey(e.target.value)}
             />
           </div>
+          <p className="login-note">
+            {key?.startsWith("nsec") && (
+              <span className="highlight">
+                We noticed that you are entering a secret key starting with nsec. We are masking your input to protect
```

Comment:
All strings should be wrapped in the `FormattedMessage` element so that we can support translations